StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/ScriptMessageHandlerWithReply.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ScriptMessageHandlerWithReply.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 5/1/24.
+//
+
+import WebKit
+
+/// Convenience class that conforms to WKScriptMessageHandlerWithReply and can be instantiated with a closure
+class ScriptMessageHandlerWithReply<T: Decodable, R: Codable>: NSObject, WKScriptMessageHandlerWithReply {
```

Comment:
nit:
- Similar comment re better type names other than `T` and `R`. Perhaps `Payload` and `Response`?
- `R` only has to be `Encodable`
```suggestion
class ScriptMessageHandlerWithReply<Payload: Decodable, Response: Encodable>: NSObject, WKScriptMessageHandlerWithReply {
```